File: DKPlugins/DKDom/DKDomHTMLSpanElement.js
// https://developer.mozilla.org/en-US/docs/Web/API/HTMLSpanElement
// https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span

var HTMLSpanElement = function HTMLSpanElement(pointer) {
 
	if(this.toString() === "[object Object]"){
		this.toString = function(){
			return "[object HTMLSpanElement]"
		}
	}
	
    return HTMLElement.call(this, pointer);
}

HTMLSpanElement.prototype = HTMLElement.prototype;

File: DKPlugins/DKDom/DKDomHTMLSpanElement.cpp
//https://developer.mozilla.org/en-US/docs/Web/API/HTMLSpanElement

#include "DK/DKApp.h"
#include "DKDom.h"
#include "DKDom/DKDomHTMLSpanElement.h"

bool DKDomHTMLSpanElement::Init(){
	DKDEBUGFUNC();
	DKClass::DKCreate("DKDom/DKDomHTMLSpanElement.js");
	return true;
}
